infrastructure/SharedInfrastructure/README.md

```diff
@@ -1,20 +1,26 @@
 
-The CDK project uses Infrastructure as Code (IaC) to define and configure the resources. It's a .NET console project, which you can examine by opening the _src/SharedInfrastructure_ project. During deployment, the CDK creates [AWS CloudFormation](https://aws.amazon.com/cloudformation) Stacks containing the resources defined using C# in the CDK application. When deploying the minimal resources needed to support the _Integrated_ launch profiles the stack is named _BookstoreIntegratedTest_. If you deploy the full resource stack equating to a "production" deployment the stack is named _BookstoreProduction_.
+The CDK project uses Infrastructure as Code (IaC) to define and configure the resources. It's a .NET console project, which you can examine by opening the _src/SharedInfrastructure_ project. During deployment, the CDK creates [AWS CloudFormation](https://aws.amazon.com/cloudformation) Stacks containing the resources defined using C# in the CDK application. The stacks in the sample application are modular and take advantage of [cross-stack references](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/walkthrough-crossstackref.html) to minimize code duplication and maximize reuse. The following stacks have been created:
 
-Before using the _Integrated_ launch profiles, you need to deploy the _BookstoreIntegratedTest_ stack contained in the CDK application. Alternatively, before deploying the admin and/or customer apps to compute services in AWS, you need to deploy the _BookstoreProduction_ stack.
+* BobsBookstoreCore - Deploys the minimal resources needed to support the _Integrated_ launch profile
+* BobsBookstoreNetwork - Deploys a VPC that is used by the application in a "production" deployment
+* BobsBookstoreDatabase - Deploys an RDS for SQL Server database that is used by the application in a "production" deployment. Depends on _BobsBookstoreNetwork_
+* BobsBookstoreEC2 - Deploys the application to an EC2 Linux instance. Depends on _BobsBookstoreCore_, _BobsBookstoreNetwork_, and _BobsBookstoreDatabase_
+* BobsBookstoreAppRunner - Deploys the application to AWS AppRunner. Depends on _BobsBookstoreCore_, _BobsBookstoreNetwork_, and _BobsBookstoreDatabase_
 
-**Note:** some of the resources created by the Production stack will incur charges to your AWS account if you are not able to take advantage of the [AWS Free Tier](https://aws.amazon.com/free).
+When deploying the minimal resources needed to support the _Integrated_ launch profile the stack is named _BobsBookstoreCore_. Alternatively you can perform a full "production" deployment to EC2 or AppRunner by deploying either the _BobsBookstoreEC2_ or _BobsBookstoreAppRunner_ stacks.
```

Comment:
Mention here of an AppRunner stack, but I don't see that stack defined in the code files?

Reply:
The comments are in anticipation of merging your AppRunner CDK stack into the codebase.